templates/NOTES.txt

```diff
@@ -2,7 +2,7 @@ Armory Remote Network Agent successfully installed or upgraded.
 
 run the following kubectl command
 
-> kubectl get pods
+> kubectl get pods -n=armory-rna
```

Comment:
Namespace is configurable -- should this file be templatized to use the configured namespace? https://helm.sh/docs/chart_template_guide/notes_files/

Reply:
Hmmm. That would be nice, but I also think its okay as is because its an example.

Reply:
Whoops, totally read that as Todd asking. 😅